src/main/java/bio/terra/service/snapshotbuilder/utils/AggregateBQQueryResultsUtils.java

```diff
@@ -1,34 +1,25 @@
 package bio.terra.service.snapshotbuilder.utils;
 
 import bio.terra.model.SnapshotBuilderConcept;
-import com.google.cloud.bigquery.TableResult;
-import java.util.List;
-import java.util.stream.StreamSupport;
+import com.google.cloud.bigquery.FieldValueList;
 
 public class AggregateBQQueryResultsUtils {
   // TODO - pull real values for hasChildren and count
-  public static List<SnapshotBuilderConcept> aggregateConceptResults(TableResult result) {
-    return StreamSupport.stream(result.iterateAll().spliterator(), false)
-        .map(
-            row -> {
-              int count;
-              try {
-                count = (int) row.get("count").getLongValue(); // If exists, use its value
-              } catch (IllegalArgumentException e) {
-                count = 1;
-              }
-              return new SnapshotBuilderConcept()
-                  .id((int) (row.get("concept_id").getLongValue()))
-                  .name(row.get("concept_name").getStringValue())
-                  .hasChildren(true)
-                  .count(count);
-            })
-        .toList();
+  public static SnapshotBuilderConcept toConcept(FieldValueList row) {
+    int count;
+    try {
+      count = (int) row.get("count").getLongValue(); // If exists, use its value
+    } catch (IllegalArgumentException e) {
+      count = 1;
```

Comment:
Will we eventually want this to throw an exception?

Reply:
The `try` `catch` statement should be removed eventually. When I was working on adding counts to the searchConcepts query, we couldn't use an alias for 'count' at the time but I think this can be removed now! I'm going to test this out and let you know.